src/clparser.cc

```diff
@@ -53,15 +53,15 @@ string CLParser::FilterShowIncludes(const string& line,
 
 // static
 bool CLParser::IsSystemInclude(string path) {
-  transform(path.begin(), path.end(), path.begin(), ::tolower);
+  transform(path.begin(), path.end(), path.begin(), [](const char c) { return static_cast<char>(::tolower(c)); });
```

Comment:
MS VC actually does have switches now https://blogs.msdn.microsoft.com/vcblog/2016/06/07/standards-version-switches-in-the-compiler/

"We’re creating a series of new switches that correspond to versions of the C++ language, starting with /std:c++14 and /std:c++latest."